scripts/src/main/resources/scripts/devon.properties

```diff
@@ -15,6 +15,8 @@ export JAVA_HOME=${DEVON_IDE_HOME}/software/java
 #export M2_REPO=~/.m2/repository
 export M2_REPO=${DEVON_IDE_HOME}/conf/.m2/repository
 export MAVEN_HOME=${DEVON_IDE_HOME}/software/maven
+# CobiGen CLI needs this
+export M2_HOME=${DEVON_IDE_HOME}/software/maven
```

Comment:
conflicts with #5 - can't you just set this variable within the `cobigen` commandlet? You may even export it there if really needed, but exporting it here will also expose it for any shell when just `devon` is called.
Further, `M2_REPO` was IMHO historically given for some reason. However, the shortcut M2 is also historically. Maven itself states on its website that M2 should be avoided and it should be `MAVEN_HOME` rahter than `M2_HOME`. Tying the version to `.m2` folder and some variable names was a kind of design flaw in maven2 and we are at maven3 already for years.

Reply:
> can't you just set this variable within the cobigen commandlet? 
I have tried to do that by adding the `export` statement to the commandlet.

However, this is not working for me because that `export` only takes effect on the subshell (please see [this explanation](https://askubuntu.com/questions/53177/bash-script-to-set-environment-variables-not-working)).

I need the `M2_HOME` variable because inside CobiGen CLI we invoke Maven in order to resolve all the CobiGen plug-ins. If it is not set, it throws this error:

```Java
java.lang.IllegalStateException: Maven application directory was not specified, and ${maven.home} is not provided in the system properties. Please specify at least on of these.
```

Maybe you can explain me how to correctly `export` the variable.